src/Accordion/AccordionItem/AccordionItem.js

```diff
@@ -24,6 +25,7 @@ class AccordionItem extends React.PureComponent {
     open: PropTypes.bool,
     disabled: PropTypes.bool,
     onToggle: PropTypes.func,
+    skin: PropTypes.oneOf(Object.values(skins)),
```

Comment:
I doesn't render properly in the documentation (storybook issue).
https://wix-wix-style-react-pr-4803.surge.sh/?activeTab=API&path=%2Fstory%2Fcomponents-api-components--accordion

It renders as: `oneOf ["Object.values(skins)"]`. As a workaround, let's write this as: 
```suggestion
    skin: PropTypes.oneOf([ 'standard' , 'light' ]),
```